cluster/src/test/java/com/hartwig/pipeline/cram/CramConversionTest.java

```diff
@@ -69,9 +69,7 @@ protected List<String> expectedCommands() {
                 input),
                 format("%s reheader --no-PG --in-place --command 'grep -v ^@PG' %s", samtools, output),
                 format("%s index %s", samtools, output),
-                format("java -Xmx4G -cp /opt/tools/bamcomp/1.3/bamcomp.jar com.hartwig.bamcomp.BamCompMain "
-                                + "-r /opt/resources/reference_genome/38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna -1 %s -2 %s -n 6 "
-                                + "--samtools-binary /opt/tools/samtools/1.14/samtools --sambamba-binary /opt/tools/sambamba/0.6" + ".8/sambamba",
+                format("java -Xmx4G -cp /opt/tools/bamcomp/1.3/bamcomp.jar com.hartwig.bamcomp.BamCompMain -r /opt/resources/reference_genome/38/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna -1 %s -2 %s -n 6 --samtools-binary /opt/tools/samtools/1.14/samtools --sambamba-binary /opt/tools/sambamba/0.6 .8/sambamba",
```

Comment:
You've got an extra space in `/opt/tools/sambamba/0.6 .8/sambamba`, not sure how this test could pass. I still see this in the latest commit.

Reply:
Actually just ran the tests locally and indeed it fails.

Reply:
Yea true, sorry that was very sloppy.